src/OpenAL/OpenToolkit.OpenAL/ALContext.cs

```diff
@@ -13,23 +14,23 @@
 
 namespace OpenToolkit.Audio.OpenAL
 {
-    public struct ALContextHandle : IEquatable<ALContextHandle>
+    public struct ALContext : IEquatable<ALContext>
     {
-        public static readonly ALContextHandle Zero = new ALContextHandle(IntPtr.Zero);
+        public static readonly ALContext Zero = new ALContext(IntPtr.Zero);
 
         public IntPtr Handle;
```

Comment:
I believe this i mentioned earlier as well, whether to expose Handle or hide it in an implicit type conversion... 

```cs
var context = new ALContext(someIntPr);
SomethingAcecptingALContext(context)

void SomethingAcceptingALContext(ALContext alContext)
{
    blablaIneedAnIntPtr(alContext);
}
```

Reply:
I actually like having the member exposed, we could go with add both?

Reply:
Would be nice if other @reviewers have something to say about this ;)

Reply:
There is an implicit cast now